docker-compose-https.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.7"
+
+services:
+
+  db:
+    image: postgres:latest
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+
+  uwsgi:
+    image: quay.io/vanessa/sregistry
+    env_file:
+      - ./.minio-env
+    restart: on-failure
+    volumes:     
+      - .:/code
+      - ./images:/var/www/images
+      - ./static:/var/www/static
+
+  nginx:
+    image: quay.io/vanessa/sregistry_nginx
+    restart: on-failure
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ./https.nginx.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./uwsgi_params.par:/etc/nginx/uwsgi_params.par:ro
+      - /etc/ssl/certs:/etc/ssl/certs:ro
+      - /etc/ssl/private:/etc/ssl/private:ro
+      - ./images:/var/www/images
+      - ./static:/var/www/static
+      - .:/code
+
+  minio:
+    image: minio/minio
+    env_file:
+      - ./.minio-env
+    ports:
+      - "9000:9000"  
+    volumes:
+      - minio_images:/images
+      - /etc/ssl/certs/chained.pem:/root/.minio/certs/public.crt:ro
+      - /etc/ssl/private/private.key:/root/.minio/certs/private.key:ro
+    command: ["server", "images"]
+
+  redis:
+    image: redis:latest
+    restart: on-failure
+
+  scheduler:
+    image: quay.io/vanessa/sregistry 
+    env_file:
+      - ./.minio-env
+    volumes:
+      - .:/code
+      - ./images:/var/www/images
+      - ./static:/var/www/static
+    command: python /code/manage.py rqscheduler
+
+  worker:
+    image: quay.io/vanessa/sregistry 
+    depends_on:
+      - uwsgi
+    env_file:
+      - ./.minio-env
+    volumes:
+      - .:/code
+      - ./images:/var/www/images
+      - ./static:/var/www/static
+    command: python /code/manage.py rqworker default
+
+volumes:
```

Comment:
Where are these volumes referenced? Typically volumes here refer to volumes created (docker) and then you would reference them in a volume definition. I think we just want the containers to bind to the host, so I'm not sure they are needed.